gdx/src/com/badlogic/gdx/maps/tiled/renderers/HexagonalTiledMapRenderer.java

```diff
@@ -109,7 +109,9 @@ private void init (TiledMap map) {
 	@Override
 	public void renderTileLayer (TiledMapTileLayer layer) {
 		final Color batchColor = batch.getColor();
-		final float color = Color.toFloatBits(batchColor.r, batchColor.g, batchColor.b, batchColor.a * layer.getOpacity());
+		final float color = Color.toFloatBits(batchColor.r * layer.getCombinedTintColor().r,
+			batchColor.g * layer.getCombinedTintColor().g, batchColor.b * layer.getCombinedTintColor().b,
+			batchColor.a * layer.getCombinedTintColor().a * layer.getOpacity());
```

Comment:
This also looks duplicate to me, but I'm not sure if it's worth it

Reply:
Yes, agreed. So there's a lot of bits of repeated code throughout all the map code code. This specifically is repeated in every single map renderer except for the OrthoCachedTiledMapRenderer.
I will be making a small change in the BatchTiledMapRenderer class which all the map renderers inherit from (except orthocached)
Which should clean this specific area and the area related to the image layer transparency stuff you commented on as well. So we don't have large chunks of duplicate code pasted across 5 renderers.